### DISCUSSION TITLE
Media Queries in CSS components

### DISCUSSION THREAD

**Discussion by fsteenkamp** [a-h/templ#273] [Q&A] [answered]
Hi, wondering if anyone has dealt with media queries in css components. I realise that the syntax completely clashes so doubt it is possible. Just curious :)

**Comment by a-h** [comment, marked as answer]
Currently, not supported, but definitely want to do something in this area. There's an issue to track it.

https://github.com/a-h/templ/issues/132

I need to catch up with @joerdav and find out what he got stuck on, and see if we can progress it.

**Reply by fsteenkamp** [reply]
Awesome thanks! This project is incredible by the way :)

**Comment by a-h** [comment]
Just a note - although it's not currently supported in CSS templ components where you can put variables in the CSS, you can use standard CSS `<style>` tags etc. and reference the class name as in normal HTML.
